frontends/web/src/routes/buy/info.tsx

```diff
@@ -59,17 +71,31 @@ class BuyInfo extends Component<Props, State> {
         }
     }
 
+    private checkSupportedCoins = () => {
+        Promise.all(
+            this.props.accounts.map((account) => (
+                apiGet(`account/${account.code}/exchange/moonpay/buy-supported`)
+                    .then(isSupported => (isSupported ? account : false))
+            ))
+        )
+        .then(results => results.filter(result => result))
+        // @ts-ignore
+        .then(accounts  => accounts.map(({ name, code }) => ({ text: name, value: code })))
+        .then(options => setTimeout(() => this.setState({ options }), 1000))
```

Comment:
hm... why timeout?

Reply:
yep, there should not be a timeout, thanks for catching